scripts/ts-bin/csvtojsonlines_test.ts

```diff
@@ -0,0 +1,34 @@
+import {csvtojsonlines} from './csvtojsonlines_lib';
+import * as stream from 'stream';
+import {expect} from 'chai';
+import 'mocha';
+
+describe('Hello function', () => {
+  it('should return hello world', async function () {
+    let r = new stream.Readable();
```

Comment:
Nit: I have a preference for more descriptive variable name.

Reply:
Ack. Me too; but when the path to usage is small, I think it's sometimes more readable to use short abbreviations.